store/store_test.go

```diff
@@ -0,0 +1,22 @@
+package store
+
+import (
+	"testing"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/stretchr/testify/require"
+)
+
+// newTestStore creates a new store for tests.
+func newTestStore(t *testing.T) *Store {
+	t.Helper()
+
+	// Create a new store.
+	db, err := sqlx.Connect("sqlite3", ":memory:")
+	require.NoError(t, err)
+
+	store := &Store{db: db}
+	store.RunMigrations()
```

Comment:
so the migrations "testing" is implicit I assume, it will fail when other tests try to import / use this.

Reply:
yes, the DB is always empty but with the right schema (latest) for tests.

If what is wrong is the sql itself, all will fail.